src/core_io.h

```diff
@@ -40,6 +41,7 @@ int ParseSighashString(const UniValue& sighash);
 
 // core_write.cpp
 UniValue ValueFromAmount(const CAmount& amount);
+UniValue ValueFromFeeRate(const CFeeRate& fee_rate);
```

Comment:
in commit 3c92185b95764f5c3536637e5923316cd9c2d8b0:

Might be good if the method name included the unit (sat/b) to avoid calling this in a btc/kb context

Reply:
Updated to allow passing in the fee mode to work with both sat/vB and BTC/kvB